browserstack.browsers.js

```diff
@@ -1,4 +1,15 @@
 const browserstackBrowsers = {
+  // iOS
+  iphone: {
```

Comment:
Yeah I couldn't find any option to hide those pink karma bars. i'll merge this for now but will keep digging in case there's a better way. 

cc @joscha in case you know how to hide those browser bars in karma.